### DISCUSSION TITLE
Faceted Search (filtering) on Stars and Lists

### DISCUSSION THREAD

**Discussion by cleder** [community/community#44984] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I just discovered `lists`, so awesome! :dark_sunglasses: 

 I have many, many projects stared, and I would like to filter my stars :star:  and lists :label:  by things like Language :snake: :gem:   and Topics :world_map: .
I could write a bot to add repositories to a list by language or topics, but before I go down this road, maybe it is under consideration?

**Comment by faeb187** [comment]
Thank you. Same here. Want to check on some repo codes and have to scroll around in the starred list, favorites reached the healthy limit too.

I think letting users create lists (like Youtube-Playlists) is a very nice-to-have, but what about:

**Tagging Repos**

no repo tags, but repo tags by user, then lists are created tagbased.

Side-effect, you get some useful statistics:
- the more common tags a repo gets the better it can be categorized and GitHub can most accurately propose "Repos of Interest" etc.

Tags can also be weighted, e.g. by colors, therefore another dimension to filter by.

Ah. Well, reusing any available filter hook would do the job I guess. 🙂 Thank you.

BTW: can't await "projects" on mobile device. You're doing great, 

Credits to all collaborators.

**Comment by slmkhanahmed** [comment]
https://addons.mozilla.org/en-US/firefox/addon/git-star/
